src/components/ebay-menu-button/index.marko

```diff
@@ -52,6 +52,7 @@ $ debugger;
         aria-label=input.a11yText
         aria-labelledby=(labelId && `${input.prefixId} ${labelId}`)
         disabled=input.disabled
+        noToggleIcon=input.noToggleIcon
```

Comment:
Maybe call this `no-icon` or `text-only`? (We have icon-only).
The main change I want is to have it support dashes. (You need to update the marko.json too)

Reply:
okay if we change the name do we need to add it to the migrations? I think `noToggleIcon` already existed on the menu-button

Reply:
Okay, leave it then. I think it's fine. 
Just make dashes supported.

Reply:
`  "@no-toggle-icon": "boolean",` is what is in marko-tag.json. Is that adequate?

Reply:
Yep!